Sources/VaporForms/Fieldset.swift

```diff
@@ -100,6 +104,11 @@ public struct Fieldset {
     return validate(content)
   }
 
+  private mutating func checkIfFieldIsRequired(_ fieldName: String){
```

Comment:
As is traditional, I have to nit-pick. First, space before final `{` please. Also, this method name doesn't sound very 'Swifty'. How about `performRequiredFieldCheck(for fieldName: String)`?